crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -2777,32 +2806,33 @@ impl<Codec: RawEntryCodec> StateMachine<Codec> {
 
     fn send_ingress_response<State>(
         ctx: &mut StateMachineApplyContext<'_, State>,
-        ingress_response: IngressResponseEnvelope<ingress::InvocationResponse>,
+        target_node: GenerationalNodeId,
```

Comment:
For what do we need the `target_node`? Wouldn't this be encoded in the `awaiting_rpcs` map in the `LeaderState`?